elements/pfe-badge/src/pfe-badge.js

```diff
@@ -0,0 +1,54 @@
+import PFElement from "../../pfelement/dist/pfelement.js";
+
+class PfeBadge extends PFElement {
+  static get tag() {
+    return "pfe-badge";
+  }
+
+  get templateUrl() {
+    return "pfe-badge.html";
+  }
+
+  get styleUrl() {
+    return "pfe-badge.scss";
+  }
+
+  static get observedAttributes() {
+    return ["number", "text", "pfe-state", "pfe-threshold"];
+  }
+
+  get threshold() {
+    return this.getAttribute('pfe-threshold');
+  }
+
+  constructor() {
+    super(PfeBadge);   
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+  }
+
+  attributeChangedCallback(attr, oldVal, newVal) {   
+    switch(attr) {
+      case "pfe-threshold":
+        this.textContent = this.threshold < this.textContent ? `${this.threshold}+` : this.textContent;
+        break;
+      case "number":
+        this.textContent = this.threshold && Number(this.threshold) < Number(newVal) ? `${this.threshold}+` : newVal;
+        break;
+      case "text":      
+        if (this.threshold) {
+          console.warn("`pfe-threshold` attribute shouldn't be used with `text` value");
+        }
+        this.textContent = newVal;
+        break;
+      default:
+        return;
+    }
+  }
```

Comment:
Maybe instead of using a slot in the HTML, we update the textContent of the span in the template and then do something like `this.shadowRoot.querySelector("span").textContent = this.textContent`.

Reply:
Made this change